app/src/main/java/br/com/catbag/gifreduxsample/models/Gif.java

```diff
@@ -14,18 +14,15 @@
         PAUSED, LOOPING, DOWNLOADING, DOWNLOADED, NOT_DOWNLOADED, DOWNLOAD_FAILED
     }
 
-    @Value
     public abstract String getUuid();
 
     @Value.Default
     public String getPath() {
         return "";
     }
 
-    @Value
     public abstract String getUrl();
-
-    @Value
```

Comment:
I guess that uses @Value.Immutable let more explicit the immutability from those methods.